CenterForOpenScience/ember-osf-web
# [EMB-566] Add editable institutions widget

## Purpose

Add editable institutions widget to registries overview page

## Summary of Changes

- Updating existing `institutions-widget` to suit our use case
- Update handbook component

## Side Effects

## Feature Flags

`ember_registries_detail_page`

## QA Notes
- Permissions: Check non-admin users do not see the edit button, admin users should.
- When the edit modal shows, make sure all user institutions are listed. And the ones already added all show on the side metadata.
- Please check the adding and deleting works without errors. And as you delete/add an institution verify the list of institutions on side metadata updates accordingly.

## Ticket

[[EMB-566]](https://openscience.atlassian.net/browse/EMB-566)

# Reviewer Checklist
